src/app/model/model.ts

```diff
@@ -0,0 +1,7 @@
+import { autoserialize } from 'cerialize';
+
+import { v4 as uuid } from 'uuid';
+
+export class Model {
```

Comment:
hie würd äuä es interface längä.

Reply:
joar.. chame schuesch mau aendere.

Reply:
I don't think an interface works here with the generic type inheritance. I would leave this a class.